Geometry_Engine/Query/Normal.cs

```diff
@@ -97,17 +107,20 @@ public static List<Vector> Normals(this Mesh3D mesh)
         /**** Public Methods - Curves                   ****/
         /***************************************************/
 
+        [Description("Returns the normal vector to the curve's plane, oriented according to the rigth hand rule. Works only for closed and planar curves.")]
+        [Input("curve", "The Polyline to get the normal to.")]
+        [Input("tolerance", "Distance tolerance used in geometry processing, default set to BH.oM.Geometry.Tolerance.Distance.")]
```

Comment:
Not sure if we should refer to `BH.oM.Geometry.Tolerance.Distance` or to 1e-6? (refers to all methods that have `tolerance` parameter)

Reply:
Had the same thought. It keeps as on the safe side, but isn't very useful for the end user.
It might a bit out of scope here but worth discussing anyway.
Having a permanent value in a string seems.. wrong. Maybe when we'll have the tolerance object we could somehow access it from the descriptions to set it dynamically?

Reply:
Maybe we could consider `typeof(ToleranceAttribute)` that would extract the actual value when `BH.Engine.Reflection.Query.Description` gets called? @al-fisher @IsakNaslundBh

Reply:
Convention is to use `BH.oM.Geometry.Tolerance.Distance` because that's what the actual default is set to. If we change the value of that tolerance in the oM, then we would have to go through and update all of the descriptions if we used the value `1e-6`.

Hovering over the input will also give the value (at least in Grasshopper, dunno about Dynamo/Excel) when it is an optional input.